examples/api/python/pythonic/README.md

```diff
@@ -0,0 +1,10 @@
+# Notes
+
+These files are copied from [the pythonic API's](TODO).
```

Comment:
Should this be the URL of the pythonic API repo?

Reply:
Yes, good catch.